src/nxdoc/nuxeo-add-ons/amazon-s3-online-storage.md

```diff
@@ -222,8 +222,8 @@ With S3 you have the option of storing your data encrypted using [S3 Client-Side
 
 The S3 Binary Manager can use a keystore containing a keypair, but there are a few caveats to be aware of:
 
-*   The Sun/Oracle JDK doesn't always allow the AES256 cipher which the AWS SDK uses internally. Depending on the US export restrictions for your country, you may be able to modify your JDK to use AES256 by installing the "Java Cryptography Extension Unlimited Strength Jurisdiction Policy Files". See the following link to download the files and installation instructions:
-    [http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html)
+*   The Azul's Zulu OpenJDK doesn't always allow the AES256 cipher which the AWS SDK uses internally. Depending on the US export restrictions for your country, you may be able to modify your JDK to use AES256 by installing the "Java Cryptography Extension Unlimited Strength Jurisdiction Policy Files". See the following link to download the files and installation instructions:
+    [https://www.azul.com/products/zulu-and-zulu-enterprise/zulu-cryptography-extension-kit/](https://www.azul.com/products/zulu-and-zulu-enterprise/zulu-cryptography-extension-kit/)
```

Comment:
I think this doesn't apply to JDK 11 anymore, for instance Oracle says https://www.oracle.com/java/technologies/javase/jdk11-readme.html#jce

Reply:
It seems to be an Oracle specificity, Zulu requires to download and install two JAR files to enable the unlimited aspect.